### PR TITLE
tests, integration-tests: port interfaces cli to spread

### PR DESCRIPTION
**Pull request by fgimenez**
Requires #1367

### CODE REVIEW

**Comment by mvo5** [comment]
:+1:

File: tests/interfaces-cli/task.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+summary: Check the interfaces command
+
+environment:
+  SNAP_NAME: network-consumer
+  SNAP_FILE: "./$[SNAP_NAME]_1.0_all.snap"
+  PLUG: network
+
+prepare: |
+  echo "Given a snap with the $PLUG plug is installed"
+  snapbuild ../lib/snaps/$SNAP_NAME .
```
Comment (kyrofa) [inline tests/interfaces-cli/task.yaml]: The network-consumer snap isn't in here-- note the test failure.

Comment (fgimenez) [inline tests/interfaces-cli/task.yaml]: @kyrofa yeap, this PR requires #1367, which carries network-consumer, once that's in this should be fine

Thanks!

Comment (kyrofa) [inline tests/interfaces-cli/task.yaml]: Haha, okay, tests are failing on master now, caught me by surprise.

**Pull request merged**